packages/tokamak/contracts-bedrock/src/L1/L1StandardBridge.sol

```diff
@@ -310,11 +358,11 @@ contract L1StandardBridge is StandardBridge, ISemver {
         internal
         override
     {
-         if (l1TONAddress == _localToken) {
+        if (l1TONAddress == _localToken) {
```

Comment:
Thank you!
I understood that Suah requested to block TON deposit in case of depositERC20. Is it right what I understood?

Reply:
Oh, he said Contract team will upgrade it. I'm sorry. I will check it resolve! 🙇🏼‍♂️